vividus-plugin-ssh/src/main/java/org/vividus/ssh/SingleCommand.java

```diff
@@ -24,7 +24,7 @@
 public class SingleCommand<T>
 {
     private final T command;
-    private final List<String> parameters;
+    private final List<Object> parameters;
```

Comment:
this change is very unsafe, in future it may lead to possible unexpected errors at runtime only, I recommend to create a new command, e.g.

```java
PUT_FROM_FILE
{
    @Override
    protected String execute(ChannelSftp channel, String filePath, String destination)
...
```